pytopo/awg_sequencing/awg_tools.py

```diff
@@ -0,0 +1,19 @@
+import time
+
+from qcodes.instrument_drivers.tektronix.AWG5014 import Tektronix_AWG5014
+from qcodes.instrument_drivers.tektronix.AWG5208 import AWG5208
```

Comment:
what about covering all children on AWG7000A (or whatever the name is) class at once? :)